src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -104,12 +105,24 @@ public boolean isAllowSchemaRelaxation() {
   public void validate(FailureCollector collector) {
     super.validate(collector);
     String bucket = getBucket();
-    if (!containsMacro(NAME_BUCKET) && bucket != null) {
-      // Basic validation for allowed characters as per https://cloud.google.com/storage/docs/naming
-      Pattern p = Pattern.compile("[a-z0-9._-]+");
-      if (!p.matcher(bucket).matches()) {
-        collector.addFailure("Bucket must only contain lowercase characters, numbers, '.', '_', and '-'", null)
-          .withConfigProperty(NAME_BUCKET);
+    // Allowed character validation for bucket name as per https://cloud.google.com/storage/docs/naming
+    String errorMessage = "Bucket name can contain following characters: lowercase letters, numbers, " +
```

Comment:
It's sounds more natural to remove 'following characters:' and say:

Bucket name can only contain lowercase letters, numbers, ...

similar comment for the other messages.